src/dynimage.rs

```diff
@@ -1109,28 +1100,37 @@ mod test {
         let check = |bit_depth, w, h, from, to| {
             assert_eq!(
                 super::gray_to_luma8(bit_depth, w, h, from).map(super::GrayImage::into_raw),
-                Some(to));
+                Some(to)
+            );
         };
         // Bit depth 1, skip is more than half a byte
         check(
-            1, 10, 2,
+            1,
+            10,
+            2,
             &[0b11110000, 0b11000000, 0b00001111, 0b11000000],
-            vec![255, 255, 255, 255, 0, 0, 0, 0, 255, 255, 0, 0, 0, 0, 255, 255, 255, 255, 255, 255]);
+            vec![
+                255, 255, 255, 255, 0, 0, 0, 0, 255, 255, 0, 0, 0, 0, 255, 255, 255, 255, 255, 255,
+            ],
```

Comment:
Ugh. Somewhat.

Reply:
Probably the line is too long for rustfmt. We can add config `max_width`.

Reply:
Yep, exactly 99 characters right now. It's even an understandable decision by rustfmt to try and wrap here but it is annoying that it does not allow any arguments to be grouped. Is there an option to allow that, as long as the line width is not exceeded? Then this would best be formatted in groups of four to show the rgb pixels.

Reply:
In imageproc, we added `#[rustfmt::skip]` and made it grid style. Should we do the same?

Reply:
Probably. I'll see if there is an upstream issue for `rustfmt` since I now encountered that problem in at least 5 projects, in different forms.